spring-cloud-huawei-governance/src/main/java/com/huaweicloud/governance/authentication/securityPolicy/SecurityPolicyAccessController.java

```diff
@@ -72,15 +69,12 @@ private boolean checkAllowAndDeny(String serviceName, AuthRequestExtractor extra
     if (securityPolicyProperties.matchAllow(serviceName, extractor.uri(), extractor.method())) {
       return !checkDeny(serviceName, extractor);
     } else {
+      // both permissive and enforcing model need print logs(send alarm info).
+      LOGGER.info("[autoauthz unauthorized request] consumer={}, provider={}, path={}, method={}, timestamp={}",
```

Comment:
same as above. Logging level shoud be WARN and message should be "request denied ...."

Reply:
fixed